cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/deploy/pipeline/ApplicationDeployable.java

```diff
@@ -34,17 +36,28 @@
   private final ApplicationSpecification specification;
   private final ApplicationSpecification existingAppSpec;
   private final ApplicationDeployScope applicationDeployScope;
+  @SerializedName("owner.principal")
```

Comment:
Yes I agree. It should be principal. Since this is used at various places (dataset  and streams which are not in this PR) and will affect some unit tests too I will like to do it in an independent PR followed just after this one if that's all right ?